site/docusaurus.config.js

```diff
@@ -46,6 +46,7 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      image: 'img/meta/open-graph.png',
```

Comment:
```suggestion
      image: '/very_good_cli/img/meta/open-graph.png',
```

don't we need to `/very_good_cli/` prefix?